components/bluetooth_proxy.rst

```diff
@@ -45,6 +45,12 @@ Configuration:
 
 - **active** (*Optional*, boolean): Enables proxying active connections. Defaults to ``false``.
 - **cache_services** (*Optional*, boolean): Enables caching GATT services in NVS flash storage which significantly speeds up active connections. Defaults to ``true`` when using the ESP-IDF framework.
+- **connection_slots** (*Optional*, int): The maximum BLE connections slots to use.
+  Each configured slot consumes ~1KB of RAM. This can only be adjusted when using
+  the ``esp-idf`` framework up to a maximum of ``9``. It is recommended not to exceed ``5``
+  connections slots to avoid memory issues. Defaults to ``3``.
```

Comment:
```suggestion
- **connection_slots** (*Optional*, int): The maximum number of BLE connection slots to use.
  Each configured slot consumes ~1KB of RAM. This can only be adjusted when using
  the ``esp-idf`` framework up to a maximum of ``9``. It is recommended not to exceed ``5``
  connection slots to avoid memory issues. Defaults to ``3``.
```